WebService/appsettings.ini

```diff
@@ -379,7 +379,7 @@ master_lock_duration = 120000
 # Note that a number bigger than ~10000 can cause errors when storing partitions
 # in Azure Tables due to the records size limit of 1MB.
 # Default: 1000, Min: 1, Max: 10000
-max_partition_size = 5000
+max_partition_size = 2000
```

Comment:
This was changed from 1000 to 5000 by @dluc in #322. If this size is too small, it looks like we'll have too many storage operations and get throttled. If it's too large, we'll hit the bug that this is attempting to fix. It would be good to understand where the "sweet spot" is for this storage system, given that we'll be writing around this many bytes per device ID:

<guid> + . + <guid> + . + ID

Which, for a 1M simulation would be:

32 + 1 + 32 + 1 + 7 

 or 73 bytes.

For 2000 devices per partition that's 73*2K + some overhead for the meta data that we write, so let's say about 146K. We should, from there, be able to figure out the relationship between when we'll be throttled at different partition sizes.

It would be great if you could figure out what the optimal value is here.

Reply:
Updated to 3K